src/app/components/bibliography/bibliography-item/bibliography-item.component.html

```diff
@@ -0,0 +1,5 @@
+<p>
+  <em *ngFor="let biblEl of templateBiblCitData" class="biblCitation">
```

Comment:
use `keyvalue` pipe instead

Reply:
This was one of the issues that took me long time in the last lap. Is perhaps the matter that could answer the question "Why are you taking so long?" 
The solution you proposed works, using `npm run start` didn't create any problems. The flaw is when I use `npm run build: gh-pages` which reports the (only) following error:
```
ERROR in src/app/components/bibliography/bibliography-item/bibliography-item.component.html(2,7): Argument of type 'BibliographicCitation' is not assignable to parameter of type 'Map<unknown, unknown>'.
  Type 'BibliographicCitation' is missing the following properties from type 'Map<unknown, unknown>': clear, delete, forEach, get, and 8 more.
```
This is the suggested template code:
```
<em *ngFor="let biblEl of biblField | keyvalue" class="biblCitation">
    {{ biblEl.value }}
</em>
```
Have I left out any property?

Reply:
Why do you need a loop in here? Remember that you will have to handle different bibliography citation style (e.g. Chicago, APA, MLA), thus you will need to handle each property of `biblField` in a very specific way, and not just showing them, no matter which is the information order.

Reply:
The loop serves just that. That is to bring out all the complex information that a bibliographic citation can contain.